client/src/components/img-button/img-button.js

```diff
@@ -0,0 +1,62 @@
+import Component from '@/components/component';
+
+import ImgBox from '@/components/img-box/img-box';
+import DefaultImg from '@/assets/images/button-default.svg';
+
+import styles from '@/styles/components/img-button/img-button.module.scss';
+
+const MAX_IMAGE_UPLOAD = 10;
+
+export default class ImgButton extends Component {
+  constructor(props) {
+    super(props);
+
+    this.$dom = this.createDom('div', {
+      className: styles['img-button'],
+    });
+
```

Comment:
```suggestion
    if (props.buttonType === 'add') {
      this.$dom.classList.add(styles.add);
    }

```

buttonType이 add일 경우 아래의 리뷰 코드에서 차이를 주기 위해서 class를 추가했으면 합니다.